hub/api/dataset.py

```diff
@@ -261,7 +262,7 @@ def like(
     def ingest(
         src: str,
         dest: str,
-        images_compression: str,
+        images_compression: str = None,
```

Comment:
we should not use `None` here, because the user may want to actually use no compression.

Reply:
make sure to update the docstring. and make sure you mention that setting this param to "auto" means the compression will be automatically determined by the most common one

Reply:
ha! that's neat